riot_pal/serial_driver.py

```diff
@@ -48,11 +45,20 @@ def _connect(self, *args, **kwargs):
                     kwargs['port'] = os.environ['PORT']
                 else:
                     kwargs['port'] = self.DEFAULT_PORT
+
+        if 'connect_wait' not in kwargs:
+            if 'CONNECT_WAIT' in os.environ:
+                kwargs['connect_wait'] = os.environ['CONNECT_WAIT']
+            else:
+                kwargs['connect_wait'] = self.DEFAULT_CONNECT_TIMEOUT
```

Comment:
wouldn't it be nice(r) to name this `DEFAULT_CONNECT_WAIT` instead 😲

Reply:
doh!

Reply:
Done